src/main/webapp/WEB-INF/views/loadCalendar.jsp

```diff
@@ -29,8 +30,8 @@
                         <div class="card shadow mb-4">
                             <div class="card-header py-3">
                                 <div class="list-group">
-                                    <c:forEach items="${childList}" var="child">
-                                        <button type="button" class="list-group-item list-group-item-action">${child.name} (born: ${child.birthDate})</button>
+                                    <c:forEach items="${childList}" var="c">
+                                        <button type="button" class="list-group-item list-group-item-action" onClick="window.location='<c:url value="/selector/${c.id}"/>'">${c.name} (born: ${c.birthDate})</button>
```

Comment:
Przyciski służące do przekierowywania powinny być deklarowane jako anchor (czyli <a>), a nie <button>. <button> nie określa, że coś wygląda jak guzik, to się ustala CSSem.